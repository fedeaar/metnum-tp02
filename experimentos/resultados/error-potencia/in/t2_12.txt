7.942248000000000000e+06 2.192440000000000000e+06 -3.857420000000000000e+05 -3.732623000000000000e+06 2.122929000000000000e+06 -2.718250000000000000e+05 1.588967000000000000e+06 1.958170000000000000e+06 1.739445000000000000e+06 -1.990939000000000000e+06 -1.944795000000000000e+06 4.707400000000000000e+05 -1.688502000000000000e+06 7.178900000000000000e+05 -8.152960000000000000e+05 -1.633594000000000000e+06 -3.269400000000000000e+04 -1.377493000000000000e+06 -9.644300000000000000e+04 -3.661094000000000000e+06
2.192440000000000000e+06 5.892105000000000000e+06 1.942594000000000000e+06 -1.190185000000000000e+06 1.296805000000000000e+06 1.706440000000000000e+06 -2.389940000000000000e+05 -6.285810000000000000e+05 3.926740000000000000e+05 -9.203050000000000000e+05 -1.485888000000000000e+06 3.899350000000000000e+05 -1.116391000000000000e+06 6.654370000000000000e+05 -1.157505000000000000e+06 3.150490000000000000e+05 -6.849600000000000000e+05 2.324470000000000000e+05 -9.727520000000000000e+05 -2.321305000000000000e+06
-3.857420000000000000e+05 1.942594000000000000e+06 8.539533000000000000e+06 2.414160000000000000e+05 4.355220000000000000e+05 -3.280330000000000000e+05 -8.870850000000000000e+05 1.550992000000000000e+06 -6.669600000000000000e+04 1.289407000000000000e+06 1.021111000000000000e+06 2.517005000000000000e+06 1.321400000000000000e+06 1.126708000000000000e+06 -2.634371000000000000e+06 2.200341000000000000e+06 7.163800000000000000e+05 1.944171000000000000e+06 -3.866090000000000000e+05 -1.381332000000000000e+06
-3.732623000000000000e+06 -1.190185000000000000e+06 2.414160000000000000e+05 7.763267000000000000e+06 -2.353210000000000000e+05 1.325909000000000000e+06 3.765300000000000000e+05 1.267321000000000000e+06 -8.716560000000000000e+05 2.385300000000000000e+04 6.173530000000000000e+05 -1.223180000000000000e+06 3.068140000000000000e+06 -5.108170000000000000e+05 1.331067000000000000e+06 4.371089000000000000e+06 1.940433000000000000e+06 1.187283000000000000e+06 1.558027000000000000e+06 1.825429000000000000e+06
2.122929000000000000e+06 1.296805000000000000e+06 4.355220000000000000e+05 -2.353210000000000000e+05 9.007744000000000000e+06 1.399909000000000000e+06 2.653402000000000000e+06 -5.973630000000000000e+05 1.303185000000000000e+06 -6.100750000000000000e+05 -2.035470000000000000e+05 -2.424438000000000000e+06 -3.917572000000000000e+06 8.512300000000000000e+05 -8.693700000000000000e+04 1.828344000000000000e+06 9.832000000000000000e+03 -7.240930000000000000e+05 2.336136000000000000e+06 -3.337696000000000000e+06
-2.718250000000000000e+05 1.706440000000000000e+06 -3.280330000000000000e+05 1.325909000000000000e+06 1.399909000000000000e+06 4.002163000000000000e+06 1.663130000000000000e+05 1.848450000000000000e+05 -8.053270000000000000e+05 -1.114576000000000000e+06 -3.725800000000000000e+05 -5.466090000000000000e+05 -2.173753000000000000e+06 2.115224000000000000e+06 -2.921360000000000000e+05 9.985480000000000000e+05 -4.789230000000000000e+05 6.102290000000000000e+05 -7.482890000000000000e+05 -2.294790000000000000e+05
1.588967000000000000e+06 -2.389940000000000000e+05 -8.870850000000000000e+05 3.765300000000000000e+05 2.653402000000000000e+06 1.663130000000000000e+05 7.423579000000000000e+06 2.040163000000000000e+06 1.895094000000000000e+06 -2.848070000000000000e+05 -1.068481000000000000e+06 -8.009710000000000000e+05 -7.878000000000000000e+05 1.464276000000000000e+06 3.327800000000000000e+04 1.925858000000000000e+06 -4.494660000000000000e+05 -2.515923000000000000e+06 2.050491000000000000e+06 -1.658862000000000000e+06
1.958170000000000000e+06 -6.285810000000000000e+05 1.550992000000000000e+06 1.267321000000000000e+06 -5.973630000000000000e+05 1.848450000000000000e+05 2.040163000000000000e+06 7.611270000000000000e+06 -3.095140000000000000e+05 -2.224995000000000000e+06 -3.846490000000000000e+05 6.866120000000000000e+05 3.238475000000000000e+06 1.740078000000000000e+06 -2.990930000000000000e+05 3.544750000000000000e+05 1.721870000000000000e+06 7.087230000000000000e+05 -1.444507000000000000e+06 1.276527000000000000e+06
1.739445000000000000e+06 3.926740000000000000e+05 -6.669600000000000000e+04 -8.716560000000000000e+05 1.303185000000000000e+06 -8.053270000000000000e+05 1.895094000000000000e+06 -3.095140000000000000e+05 4.353965000000000000e+06 -1.274953000000000000e+06 9.438800000000000000e+04 1.310630000000000000e+05 -1.474934000000000000e+06 6.863930000000000000e+05 -8.452840000000000000e+05 -5.157910000000000000e+05 -1.964308000000000000e+06 -1.921398000000000000e+06 9.890230000000000000e+05 -1.076800000000000000e+06
-1.990939000000000000e+06 -9.203050000000000000e+05 1.289407000000000000e+06 2.385300000000000000e+04 -6.100750000000000000e+05 -1.114576000000000000e+06 -2.848070000000000000e+05 -2.224995000000000000e+06 -1.274953000000000000e+06 8.434652000000000000e+06 1.614544000000000000e+06 -6.861380000000000000e+05 2.715880000000000000e+05 -2.273997000000000000e+06 -5.730740000000000000e+05 1.431808000000000000e+06 8.958160000000000000e+05 6.687160000000000000e+05 2.363340000000000000e+06 -1.519184000000000000e+06
-1.944795000000000000e+06 -1.485888000000000000e+06 1.021111000000000000e+06 6.173530000000000000e+05 -2.035470000000000000e+05 -3.725800000000000000e+05 -1.068481000000000000e+06 -3.846490000000000000e+05 9.438800000000000000e+04 1.614544000000000000e+06 4.182545000000000000e+06 5.106400000000000000e+05 2.933190000000000000e+05 1.394530000000000000e+05 -7.455600000000000000e+04 1.025902000000000000e+06 -1.271905000000000000e+06 1.301354000000000000e+06 -9.942970000000000000e+05 -1.872600000000000000e+05
4.707400000000000000e+05 3.899350000000000000e+05 2.517005000000000000e+06 -1.223180000000000000e+06 -2.424438000000000000e+06 -5.466090000000000000e+05 -8.009710000000000000e+05 6.866120000000000000e+05 1.310630000000000000e+05 -6.861380000000000000e+05 5.106400000000000000e+05 7.289695000000000000e+06 2.835524000000000000e+06 -5.985520000000000000e+05 -1.122191000000000000e+06 1.481710000000000000e+05 -3.292440000000000000e+06 3.789457000000000000e+06 -1.349099000000000000e+06 4.528200000000000000e+05
-1.688502000000000000e+06 -1.116391000000000000e+06 1.321400000000000000e+06 3.068140000000000000e+06 -3.917572000000000000e+06 -2.173753000000000000e+06 -7.878000000000000000e+05 3.238475000000000000e+06 -1.474934000000000000e+06 2.715880000000000000e+05 2.933190000000000000e+05 2.835524000000000000e+06 6.937661000000000000e+06 -2.496188000000000000e+06 1.432241000000000000e+06 1.934564000000000000e+06 1.628976000000000000e+06 2.320561000000000000e+06 -6.868340000000000000e+05 2.767854000000000000e+06
7.178900000000000000e+05 6.654370000000000000e+05 1.126708000000000000e+06 -5.108170000000000000e+05 8.512300000000000000e+05 2.115224000000000000e+06 1.464276000000000000e+06 1.740078000000000000e+06 6.863930000000000000e+05 -2.273997000000000000e+06 1.394530000000000000e+05 -5.985520000000000000e+05 -2.496188000000000000e+06 7.171288000000000000e+06 1.517380000000000000e+05 -3.290750000000000000e+05 -3.497977000000000000e+06 -1.184327000000000000e+06 -1.566713000000000000e+06 -1.642522000000000000e+06
-8.152960000000000000e+05 -1.157505000000000000e+06 -2.634371000000000000e+06 1.331067000000000000e+06 -8.693700000000000000e+04 -2.921360000000000000e+05 3.327800000000000000e+04 -2.990930000000000000e+05 -8.452840000000000000e+05 -5.730740000000000000e+05 -7.455600000000000000e+04 -1.122191000000000000e+06 1.432241000000000000e+06 1.517380000000000000e+05 5.466555000000000000e+06 9.922210000000000000e+05 4.949030000000000000e+05 -1.842478000000000000e+06 -1.704132000000000000e+06 8.732430000000000000e+05
-1.633594000000000000e+06 3.150490000000000000e+05 2.200341000000000000e+06 4.371089000000000000e+06 1.828344000000000000e+06 9.985480000000000000e+05 1.925858000000000000e+06 3.544750000000000000e+05 -5.157910000000000000e+05 1.431808000000000000e+06 1.025902000000000000e+06 1.481710000000000000e+05 1.934564000000000000e+06 -3.290750000000000000e+05 9.922210000000000000e+05 6.178055000000000000e+06 1.085502000000000000e+06 4.528290000000000000e+05 8.178950000000000000e+05 -3.006950000000000000e+05
-3.269400000000000000e+04 -6.849600000000000000e+05 7.163800000000000000e+05 1.940433000000000000e+06 9.832000000000000000e+03 -4.789230000000000000e+05 -4.494660000000000000e+05 1.721870000000000000e+06 -1.964308000000000000e+06 8.958160000000000000e+05 -1.271905000000000000e+06 -3.292440000000000000e+06 1.628976000000000000e+06 -3.497977000000000000e+06 4.949030000000000000e+05 1.085502000000000000e+06 6.863360000000000000e+06 -9.049390000000000000e+05 8.520440000000000000e+05 1.725421000000000000e+06
-1.377493000000000000e+06 2.324470000000000000e+05 1.944171000000000000e+06 1.187283000000000000e+06 -7.240930000000000000e+05 6.102290000000000000e+05 -2.515923000000000000e+06 7.087230000000000000e+05 -1.921398000000000000e+06 6.687160000000000000e+05 1.301354000000000000e+06 3.789457000000000000e+06 2.320561000000000000e+06 -1.184327000000000000e+06 -1.842478000000000000e+06 4.528290000000000000e+05 -9.049390000000000000e+05 5.305924000000000000e+06 3.012810000000000000e+05 7.438290000000000000e+05
-9.644300000000000000e+04 -9.727520000000000000e+05 -3.866090000000000000e+05 1.558027000000000000e+06 2.336136000000000000e+06 -7.482890000000000000e+05 2.050491000000000000e+06 -1.444507000000000000e+06 9.890230000000000000e+05 2.363340000000000000e+06 -9.942970000000000000e+05 -1.349099000000000000e+06 -6.868340000000000000e+05 -1.566713000000000000e+06 -1.704132000000000000e+06 8.178950000000000000e+05 8.520440000000000000e+05 3.012810000000000000e+05 5.838916000000000000e+06 -1.089258000000000000e+06
-3.661094000000000000e+06 -2.321305000000000000e+06 -1.381332000000000000e+06 1.825429000000000000e+06 -3.337696000000000000e+06 -2.294790000000000000e+05 -1.658862000000000000e+06 1.276527000000000000e+06 -1.076800000000000000e+06 -1.519184000000000000e+06 -1.872600000000000000e+05 4.528200000000000000e+05 2.767854000000000000e+06 -1.642522000000000000e+06 8.732430000000000000e+05 -3.006950000000000000e+05 1.725421000000000000e+06 7.438290000000000000e+05 -1.089258000000000000e+06 5.107942000000000000e+06
