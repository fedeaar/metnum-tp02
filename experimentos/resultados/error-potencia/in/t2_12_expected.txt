2.414858396590259299e+07
1.752553762861238420e+07
1.545110238391550444e+07
1.472581899878376536e+07
1.174098913708437048e+07
9.519472830508543178e+06
7.616490765337545425e+06
6.810192078778170981e+06
5.552639044810746796e+06
5.292642271910876036e+06
3.665796159151212312e+06
3.219156012387222145e+06
2.349766889962324873e+06
1.667556298289457336e+06
1.046034639890913386e+06
6.837132088653296232e+05
1.587943353943416732e+05
1.101645865512481250e+05
2.712113705550618397e+04
8.946268080547635009e+02
